dhall/src/Dhall.hs

```diff
@@ -1044,9 +1048,14 @@ data InterpretOptions = InterpretOptions
 -}
 defaultInterpretOptions :: InterpretOptions
 defaultInterpretOptions = InterpretOptions
-    { fieldModifier       = id
-    , constructorModifier = id
-    , inputNormalizer     = Dhall.Core.ReifiedNormalizer (const (pure Nothing))
+    { fieldModifier =
+          id
+    , constructorModifier =
+          id
+    , collapseSingletonRecords =
+          False
```

Comment:
If the option could differentiate between named and anonymous fields, I think the default should be to collapse records of anonymous fields, but to preserve named fields.

If we can't differentiate, I think the default should be `True`, since that is what was requested in https://github.com/dhall-lang/dhall-haskell/issues/346#issuecomment-520123686 and at least [one user didn't seem to mind](https://github.com/dhall-lang/dhall-haskell/pull/1315#issuecomment-531878691).